bin/tflocal

```diff
@@ -118,6 +120,10 @@ def create_provider_config_file(provider_aliases=None):
     for provider in provider_aliases:
         endpoints = "\n".join([f'{s} = "{get_service_endpoint(s)}"' for s in services])
         provider_config = TF_PROVIDER_CONFIG.replace("<endpoints>", endpoints)
+        provider_config = provider_config.replace(
+            "<access_key>",
+            get_access_key(provider) if CUSTOMISE_ACCESS_KEY else DEFAULT_ACCESS_KEY
+        )
```

Comment:
this is a extremely small nit, but I would replace the `access_key` before replacing the `endpoints`, as it makes the string much bigger once the endpoints are in, and the search to replace would be longer. with a quick `timeit` checks, it seems it'd be around 30% slower.
```suggestion
        provider_config = TF_PROVIDER_CONFIG.replace(
            "<access_key>",
            get_access_key(provider) if CUSTOMISE_ACCESS_KEY else DEFAULT_ACCESS_KEY
        )
        endpoints = "\n".join([f'{s} = "{get_service_endpoint(s)}"' for s in services])
        provider_config = provider_config.replace("<endpoints>", endpoints)

```

We actually should be able to move the list comprehension to get the endpoints out of the loop and use it inside the loop, to not recalculate it every time when the result would be the same. 

Nothing blocking, this can be merged as is!